pkg/utils/flagbuilder.go

```diff
@@ -0,0 +1,53 @@
+package utils
```

Comment:
I think the validators module would be a good home for this instead of generic utils if the intent is for flag builder to be used for validator clients. 

Or could do in package called "software" (or something like that) and we use flagbuilder for all software we start.